spec/requests/sights_spec.rb

```diff
@@ -1,12 +1,157 @@
 require 'rails_helper'
 
 RSpec.describe "Sights", type: :request do
-  # -----index-----
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
-  end
-
-# -----create-----
-# -----update-----
-# -----delete-----
+    let(:user) { User.create email: 'kevin@testing.com', password: 'testing123', 
+    password_confirmation: 'testing123'}
+
+    let(:trip1) { Trip.create trip_name: 'Getting some sun', trip_location: "Jamaica", trip_date_range: "12/20/2021 - 1/5/2022", is_public: false, user_id: user.id}
+
+    describe "POST /sights" do
+      it 'creates a new sight' do
+        sight_params = {
+          sight: {
+            name: "Taco Bell",
+            address1: "Bell Street",
+            city: "Taco",
+            zip_code: "11111",
+            state: "Jamaica Land",
+            country: "Jamaica",
+            phone: "222-222-2222",
+            trip_id: trip1.id
+          }
+        }
+        post '/sights', params: sight_params
+
+        sight_response = JSON.parse(response.body)
+        expect(sight_response['name']).to eq 'Taco Bell'
+        expect(sight_response['address1']).to eq 'Bell Street'
+        expect(sight_response['city']).to eq 'Taco'
+      end
+
+      it 'cannot create a new sight without a name' do
+        sight_params = {
+          sight: {
+            address1: "Bell Street",
+            city: "Taco",
+            zip_code: "11111",
+            state: "Jamaica Land",
+            country: "Jamaica",
+            phone: "222-222-2222",
+            trip_id: trip1.id
+          }
+        }
+        post '/sights', params: sight_params
+        error_response = JSON.parse(response.body)
+        expect(error_response['name']).to include "can't be blank"
+        expect(response).to have_http_status(422)
+      end
+
+      it 'cannot create a new sight without a address1' do
```

Comment:
you do not need to cover each validation error in the controller specs because that's covered in the model specs. At most, tests need to cover just one error response